Xlib/ext/randr.py

```diff
@@ -1186,6 +1269,11 @@ def init(disp, info):
     disp.extension_add_method('display', 'xrandr_get_panning', get_panning)
     disp.extension_add_method('display', 'xrandr_set_panning', set_panning)
 
+    # version 1.5 compatible
+    disp.extension_add_method('window', 'xrandr_get_monitors', get_monitors)
```

Comment:
I'm not sure if version 1.5 is present almost everywhere for now. Does it make sense to check version of the extension? So that we can support both 1.3 and 1.5+.